testER/src/App.jsx

```diff
@@ -357,32 +342,40 @@ const App = (props) => {
     // toggle off all tracks??
   }
 
-  return (
-    <>
-      <div id='map-container' onKeyDown={logKey} onKeyUp={logKey}>
-        {/* <a href='https://earthranger.com/'>
+  return <TrackContext.Provider value={{ displayTracks, setTracks, tracks }}>
+    <div id='map-container' onKeyDown={logKey} onKeyUp={logKey}>
+      {/* <a href='https://earthranger.com/'>
           <img src='./public/images/LogoEarthRanger.png' id='earth-ranger-logo' />
         </a> */}
-        <Legend
-          subs={subjects}
-          track={tracks}
```

Comment:
Note that `track={tracks}` and `onTrackClick` have been removed, as they are now provided via the context API where needed.